### PR TITLE
Cleaned a loop with for-range

### PR DESCRIPTION
**Pull request by hv7214**
Replaced a loop with for-range on L438 in *openni_tracking.cpp*

**Related Issue**: #4214

### CODE REVIEW

File: apps/src/openni_tracking.cpp [hunk 1]
```diff
@@ -435,12 +435,12 @@ class OpenNISegmentTracking {
     result.width = cloud->width;
     result.height = cloud->height;
     result.is_dense = cloud->is_dense;
-    for (std::size_t i = 0; i < cloud->size(); i++) {
+    for (auto pt : *cloud) {
```
Comment (kunaltyagi) [inline apps/src/openni_tracking.cpp:438]:
```suggestion
    for (const auto& pt : *cloud) {
```

**Review by kunaltyagi — COMMENTED** [review]
Apart from minor edit, LGTM

Use of references minimizes copies, and creates more efficient binaries

File: apps/src/openni_tracking.cpp (hunk 1 above)
Comment (hv7214) [reply apps/src/openni_tracking.cpp:438]: @kunaltyagi thanks for pointing out, will make this change.

**Review by kunaltyagi — APPROVED** [review]

**Review by mvieth — APPROVED** [review]
Thanks!

**Pull request merged**